public/js/complex-game.js

```diff
@@ -0,0 +1,273 @@
+var g = NewGame("Complex Game", {
+	version        : "0.1",
+	intro          : "Oh look!  A house!  Let's go in it!",
```

Comment:
Think we should, by default, append game function directions? or do we just want people to dig through the window, aka hard mode?